### PR TITLE
Created toolbar and headline

### PR DESCRIPTION
**Pull request by LeslieAine**
In this project feature I created a mobile website for my portfolio. I only worked on the toolbar and headline for now in the following steps;
1. Created an html file
2. Created a css file
3. Set up linters for my repo
4. Created a .gitignore file
5. Used flexbox for css

### CODE REVIEW

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @LeslieAine, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## ❌ Invalid Code Review Request

You have submitted a project, that has a failed linter check. Kindly fix your linter errors and submit again 👍🏽 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  <head>
+    <meta charset="utf-8" />
+    <title> Leslie Portfolio project</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body class="body">
+    <header class="header">
+      <nav class="navigate">
+        <a href=""><img src="images/John Doe.png" alt="" /></a>
```
Comment (brainconnect93) [inline index.html:12]: - [ ] According to the project template, the logo and the navbar are aligning in a way where there is spaces around them.
front, back, bottom and top. 👇 

compare to what you have in you project: 👇

Comment (brainconnect93) [inline index.html:12]: - [ ] According to the project template. there is a space around the logo and the nav hamburger, i.e space in the top, right, bottom, and left. As seen below: 👇 

head1

compare to what we have in your project: 👇 

cor

- Kindly check the [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=DSRLSZBE6X8joMdm-0) for proper styling of the navbar header

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  font-family: Crete Round, var(--font-family-common);
+}
+
+.describe {
+  font-family: Roboto, var(--font-family-common);
+  font-style: normal;
+  font-weight: 500;
+  font-size: 32px;
+  color: #fff;
+}
```
Comment (brainconnect93) [inline styles.css:47]: - [ ] I would suggest we stick with the Figma template. the alignment of the page title is not well positioned.
Kindly check the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=kgjXcDYS4nMWzLhZ-0) for the `padding` and `margin` to use,  as well as the `font-size` and `font-family` that easily fit into your project for proper alignment

kindly compare what we have in the Figma template: 👇 

body 1

to what we have in your project: 👇 

body

File: index.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        <a href=""><img src="images/Union.png" alt="" /></a>
+      </nav>
+    </header>
+    <div class="heading-section">
```
Comment (brainconnect93) [inline index.html:16]: - [ ]   I would suggest you kindly use a `<section>` tag to start a new section of your project which will have all other `<div>` wrapped inside it. As this represents a generic standalone section of the document. Kindly check this [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+.heading-section {
+  display: flex;
+  flex-direction: column;
+}
```
Comment (brainconnect93) [inline styles.css:14]: - [ ] According to the Figma template, the section of this project has a background image but that seems to be missing in your project. Kindly follow this [link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=kgjXcDYS4nMWzLhZ-0) of the Figma template to download the proper image and use.
- Click this [link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) to learn more about CSS-background image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+      <div class="heading1">
+        <h1 class="heading">Hey There <br />I’m Leslie</h1>
+      </div>
+      <div class="describe">
+        <h2 class="describe">I am a Software Developer</h2>
```
Comment (brainconnect93) [inline index.html:21]: - [ ] I would suggest you change this to a paragraph tag i.e `<p>`. This way will be semantically correct and the screen readers and other technologies will thank you.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+          ><img class="social-icons" src="images/linkedin.png" alt=""
+        /></a>
+        <a href=""><img class="social-icons" src="images/medium.png" alt=""
+        /></a>
+        <a href="https://github.com/LeslieAine"
+          ><img class="social-icons" src="images/github.png" alt=""
+        /></a>
+        <a href=""><img class="social-icons" src="images/angelist.png" alt=""
+        /></a>
+      </div>
```
Comment (brainconnect93) [inline index.html:42]: - [ ] I would kindly suggest you make use of a `<ul>` tag as a container to group all your icons and each icon should have an `<li>` tag with an anchor tag i.e `<a>`.  Apart from being semantically correct, the `<a>` tag will let you connect your social networks with the icons, using href.

Example: 👇 
```HTML
<ul class="social">
    <li>
        <a href="https://mobile.twitter.com/Leslie"><img src="images/twitter.png" alt=""/></a> 
    </li>
    <li>
        <a href="https://www.linkedin.com/in/LeslieAine"><img src="images/linkedin.png" alt=""/></a>
     </li>
</ul>
```

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @LeslieAine  , 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [ ]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+                        <a href="#"><img src="images/medium.png" alt=""/></a> 
+                    </li>
+                    <li class="social-icons">
+                        <a href="#"><img src="images/Angelist.png" alt=""/></a> 
+                    </li>
+                    </ul>
+    </section>
+    
+      
+        <img class="background"src="images/Group.png" alt="">
```
Comment (levy002) [inline index.html:44]: - [ ]  As requested by the previous reviewer, kindly make sure to use the CSS background property to insert the background image on your page. It doesn't look professional to add a background image by using HTML. I highly recommend you to use CSS to insert your background image. :wink:

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+  margin: 10% 0 0;
+}
+
+.describe {
+  font-family: Roboto, var(--font-family-common);
+  font-style: normal;
+  font-weight: 500;
+  font-size: 32px;
+  color: #fff;
+}
```
Comment (levy002) [inline styles.css:53]: - [ ] As requested by the previous reviewer, kindly make sure your design looks exactly like provided Figma template. Make sure your typography meets the design. In order to use these special google fonts, you must first import them at the top of your CSS file from [google fonts](https://fonts.google.com/) in order to be able to see the changes on your screen.

     _**Example of importing Poppins**_
      `@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');`
     _**and you can use it in CSS like this**_
      `font-family: 'Poppins', sans-serif;`

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+
+.headline {
+  font-family: Poppins, var(--font-family-common);
+  font-size: 20px;
+  line-height: 32px;
+  color: #fff;
+}
```
Comment (levy002) [inline styles.css:60]: - [ ] As requested by the previous reviewer, kindly make sure your typography looks exactly like the provided design. To use these special fonts, you must first import them at the top of your CSS file from [google fonts](https://fonts.google.com/) in order to use it.

     _**Example of how to import the google font**_
     `@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');`

    _**and you can use it like this you css**_
   `font-family: 'Poppins', sans-serif;`

    - ### Your design
     image

   - ### Figma design
   
    image

File: styles.css [hunk 10]
```diff
@@ -0,0 +1,80 @@
...
+
+/* This is the social styling */
+
+.social {
+  display: flex;
+}
```
Comment (levy002) [inline styles.css:66]: - [OPTIONAL] Kindly make sure to remove these dots which is appearing on all unordered list and please make sure to center your social media icons in order to meet the provided Figma design. :wink:

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @LeslieAine   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 11]
```diff
@@ -0,0 +1,90 @@
...
+body {
+  background-color: #1c1a19;
+  margin: 0;
+  height: 100%;
+  width: 100%;
+  background-image: url(images/Group.png);
+  background-repeat: no-repeat;
+  background-position-x: right;
+  background-position-y: bottom;
+}
```
Comment (Ugboaja-Uchechi) [inline styles.css:44]: - [ ] [OPTIONAL] Kindly make sure your `background-image` is aligned properly as seen in the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

**Review by mukhammadrizooff — COMMENTED** [review]
Hi @LeslieAine, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] [REQUIRED] You have submitted a project, that wrong submition link for `Portfolio: finish mobile version` milestone, please submit correct pull requist link on your dashboard 🙏 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**